samples/ChangefeedMigrationSample/Program.cs

```diff
@@ -0,0 +1,193 @@
+﻿namespace ChangefeedMigrationSample
```

Comment:
Apache 2.0 or MIT open license header? Some of the samples have MIT open source headers